### ISSUE TITLE
Wrong error msg during signup

### ISSUE THREAD

**Issue by ananyaarun** [publiclab/plots2#5115]
# Description
Similar to issue #5114 , while workin on #3340 , I found that entering a different paddword in the confirm password section when compared to the password section gives this error msg which is misleading to a common user 
![signuperror](https://user-images.githubusercontent.com/32260628/54482383-c88d1a80-4868-11e9-987a-eeebe71db509.png)

# Solution
The error msg should be changed

**Comment by ananyaarun** [comment]
I'd like to work on this. Or if the fix is easy should i make this an FTO ? @gauravano

**Comment by CleverFool77** [comment]
@ananyaarun If the password is same, Are you able to form an account in local ? I tried but I'm unable to make an account :thinking:

**Comment by ananyaarun** [comment]
@CleverFool77 , Yes i can create multiple accounts in my local by entering the right details. You must have initially created an account you r working with right now right ? What problem are u facing ? Can u add a screenshot If possible ?

**Comment by ananyaarun** [comment]
Also there are some required conditions like username should be atleast 3 characters and password must be 8 minimum. Make sure you follow these as well :)

**Comment by CleverFool77** [comment]
I did gave correct info but it still says,, you are spam and  not human :cry:

**Comment by CleverFool77** [comment]
And I'm working with admin always.

**Comment by CleverFool77** [comment]
@ananyaarun I got my mistake :).

**Comment by gauravano** [comment]
Let's do form validation first and then we can resolve this one?

**Comment by ananyaarun** [comment]
@gauravano , Sure.
Here are the PR's for login and signup form validation corrections. 
- https://github.com/publiclab/plots2/pull/5087
- https://github.com/publiclab/plots2/pull/5116
You can take a look at it whenever you are free :) Thanks!
